site/content/docs/include/http-config.en.md

```diff
@@ -42,6 +43,12 @@ The approximate amount of time, in seconds, between health checks of an individu
 <span class="parent-field">http.healthcheck.</span><a id="http-healthcheck-timeout" href="#http-healthcheck-timeout" class="field">`timeout`</a> <span class="type">Duration</span>  
 The amount of time, in seconds, during which no response from a target means a failed health check. The default is 5s. Range 5s-300s.
 
+<span class="parent-field">http.healthcheck.</span><a id="http-healthcheck-deregistration-delay" href="#http-healthcheck-deregistration-delay" class="field">`deregistration_delay`</a> <span class="type">Duration</span>  
+The amount of time to wait for targets to drain connections during deregistration. The default is 60s. Setting this to a larger value gives targets more time to gracefully drain connections, but increases the time required for new deployments.
```

Comment:
Should we mention that the range is `0-3600` seconds?

Reply:
Also, these are super helpful explanations on the two new fields! Awesome

Reply:
Great call. Fixed.